clients/admin-ui/src/features/config-wizard/OrganizationInfoForm.tsx

```diff
@@ -132,21 +131,12 @@ const OrganizationInfoForm = () => {
     >
       <Stack spacing={10}>
         <Heading as="h3" size="lg">
-          Tell us about your business
+          Create your Organization
         </Heading>
         <div>
           Provide your organization information. This information is used to
-          configure your organization in Fidesctl for{" "}
-          <Tooltip
```

Comment:
Figma still has "data map" colored blue like this tooltip is still needed. I don't know what copy it's supposed to contain, though.

Reply:
In [this issue](https://github.com/ethyca/fidesctl-plus/issues/231#issuecomment-1283995139), @mfbrown mentioned "Let's also remove the "data map" link." -- I assumed that also meant to unhighlight it as well, since the tooltip/link is being removed, but maybe @mfbrown can clarify here. 🙂

Reply:
Yes, remove the link and the special text color